static/app/components/events/eventReplay/replayPreview.tsx

```diff
@@ -50,21 +50,33 @@ function ReplayPreview({orgSlug, replaySlug, event}: Props) {
 
   if (fetchError) {
     const reasons = [
-      t('The replay is still processing'),
-      t('The replay has been deleted by a member in your organization'),
-      t('There is an internal system error'),
+      t('The replay was rate-limited and could not be accepted.'),
+      t('The replay has been deleted by a member in your organization.'),
+      t('There were network errors and the replay was not saved.'),
+      tct('[link:Read the docs] to understand why.', {
+        link: (
+          <ExternalLink href="https://docs.sentry.io/platforms/javascript/session-replay/#error-linking" />
+        ),
+      }),
     ];
 
     return (
-      <Alert type="info" showIcon data-test-id="replay-error">
+      <Alert
+        type="info"
+        showIcon
+        data-test-id="replay-error"
+        trailingItems={
+          <Button
+            size="xs"
```

Comment:
Let's make this open in a new tab?

```suggestion
            external
            size="xs"
```